petstore/servers/aspnet/generated/models/ResourceDeletedResponse.cs

```diff
@@ -1,19 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-// <auto-generated />
-
-using System;
-using System.Text.Json;
-using System.Text.Json.Serialization;
-
-namespace PetStore.Service.Models
-{
-
-    ///<summary>
-    /// Resource deleted successfully.
-    ///</summary>
-    public partial class ResourceDeletedResponse
```

Comment:
@ArcturusZhang is this expected?

Reply:
I kind of do not know.
If your test could pass, this should be fine.